docs/content/action/handler.md

```diff
@@ -1,81 +1,116 @@
 ---
-title: Handler
 sort: 2
 ---
 
-# Metadata "handler" (function, required)
+# Handler
 
-The "handler" property define the function that an action implements to get things done. It is fundamental to each action.
+The `handler` property defines the function that an action implements to get things done. It is fundamental to each action.
 
-The property is required but most of the time, you don't have to write an handler function on your own. Instead, you will use an existing action which was previously [registered](/usages/registry/).
+The property is required but most of the time, you don't have to write a handler function on your own. Instead, you can use an existing action which was previously [registered](/current/usages/registry/).
 
-However, you should not be afraid to write your own handler, it is as easy as writing a plain vanilla JavaScript function and using the Nikita `call` action to schedule its execution. Functions can run [synchronously or asynchronously](/usages/sync_async/).
+However, you should not be afraid to write your own handler, it is as easy as writing a plain vanilla JavaScript function and using the Nikita `call` action to schedule its execution. 
 
-## Synchronous handlers
+## Basic example
 
-Synchronous handlers expect one arguments, the config passed to the action:
+The [configuration properties](/current/action/config) passed to the `call` action are available in the `config` property of the first argument of the handler:
 
 ```js
-require('nikita')
+nikita
 .call({
   key: 'value',
-  handler: function({config}){
-    // do something
-    assert(config.key, 'value')
-  }
-})
-```
-
-## Asynchronous handlers
-
-Asynchronous handlers expect two arguments, the config and a callback function to be called when the action has terminated:
-
-```js
-require('nikita')
-.call({
-  key: 'value',
-  handler: function({config}, callback){
-    setImmediate(function(){
-      // do something
-      assert(config.key, 'value')
-    })
+  handler: ({config}) => {
+    // Do something
+    console.info(config.key)
   }
 })
 ```
 
 ## Style
 
-You will probably never see an handler function being defined by the "handler" config key. Instead, we define it with an alternative syntax by passing the handler function as an independent argument. The example above is preferably rewritten as:
+You will probably never see a handler function is defined by the `handler` property. Instead, we define it with an alternative syntax bypassing the handler function as an independent argument. The example above is preferably rewritten as:
```

Comment:
Not `is` but `being` or just nothing like `You will probably never see a handler function defined by...`

Reply:
`bypassing` is an entirely different meaning. It was meant to be `by passing`, you can change to `by providing` or anything of your liking but not `bypassing`.